conda-forge.yml

```diff
@@ -18,5 +18,9 @@ os_version:
 conda_build:
   pkg_format: '2'
 build_platform:
-   osx_arm64: osx_64
+  osx_arm64: osx_64
+  linux_aarch64: linux_aarch64
```

Comment:
I think this needs to be linux_64 to use cross compilation. No way we will be able to compile QGIS without it, at least because of the time it would take.

Reply:
Good catch!